addons/crm/views/crm_lead_views.xml

```diff
@@ -591,7 +591,14 @@
                                         <field name="title" placeholder="Title" domain="[]" options='{"no_open": True}'/>
                                     </div>
                                     <field name="function"/>
-                                    <field name="mobile"/>
+                                    <field name="partner_address_mobile"
+                                       attrs="{'invisible': [('partner_address_mobile', '==', False)]}"
```

Comment:
We usually use a single `=`, so `attrs="{'invisible': [('partner_address_mobile', '=', False)]}"`